test/OpenTelemetry.AutoInstrumentation.Tests/Configuration/PluginsConfigurationHelperTests.cs

```diff
@@ -87,9 +87,9 @@ public void PluginTypeMissingExpectedMethod()
 
         using (new AssertionScope())
         {
-            tracerAction.Should().Throw<MissingMethodException>();
-            meterAction.Should().Throw<MissingMethodException>();
-            logsAction.Should().Throw<MissingMethodException>();
+            tracerAction.Should().NotThrow();
```

Comment:
Perhaps a plugin that doesn't match the interface for any signal should still throw. I'm open to change that but it seems reasonable to keep throwing here.

Reply:
If we unify to single environment variable we loss an option to throw on missing method.

Reply:
I think we can still reasonably do it but it requires a refactor mentioned in other comment. Anyway, we can discuss that possibility out of this PR.